pypeit/scripts/skysub_regions.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+"""
+This script enables the user to view a 2D FITS file
+and define the sky background regions interactively.
+Run above the Science/ folder.
+"""
+
+import os
+import argparse
+import numpy as np
+from configobj import ConfigObj
+
+from astropy.table import Table
+
+from pypeit import msgs
+from pypeit import edgetrace
+from pypeit.core.gui.skysub_regions import SkySubGUI
+from pypeit.core.parse import get_dnum
+from pypeit.core import procimg
+from pypeit.par.util import parse_pypeit_file
+from pypeit.par import PypeItPar
+from pypeit.masterframe import MasterFrame
+from pypeit.spectrographs.util import load_spectrograph
+from pypeit.metadata import PypeItMetaData
+
+
+def parser(options=None):
+
+    parser = argparse.ArgumentParser(description='Display a Raw science image and interactively define'
+                                                 'the sky regions using a GUI. Run in the same folder'
+                                                 'as your .pypeit file',
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument('file', type = str, default=None, help='PypeIt file')
+    parser.add_argument('--det', default=1, type=int, help="Detector")
+
+    return parser.parse_args() if options is None else parser.parse_args(options)
+
+
+def parse_traces(hdulist_1d, det_nm):
```

Comment:
I worry about a custom reader here.
Is there a reason not to use the I/O of EdgeTrace?

Actually, I'm not sure this is even called..

Reply:
Right again - removed... this was left over from a previous version.